sql/core/src/test/scala/org/apache/spark/sql/JsonFunctionsSuite.scala

```diff
@@ -840,4 +841,29 @@ class JsonFunctionsSuite extends QueryTest with SharedSparkSession {
       }
     }
   }
+
+  test("SPARK-35094: Spark from_json(JsonToStruct) function return wrong value " +
```

Comment:
@Peng-Lei this test passes without your fix. Can you show the reproducible codes with before/after results?

Reply:
@HyukjinKwon I am so sorry, It did pass without my fix. My fix is simply skipping the parsing when an parsing exception occurs.